Samples/Samples.iOS/Samples.iOS.csproj

```diff
@@ -115,6 +115,9 @@
     <ErrorReport>prompt</ErrorReport>
     <CodeAnalysisRuleSet>C:\GitHub\xamarin\Essentials\Xamarin.Essentials.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(RunConfiguration)' == 'Default' ">
```

Comment:
We should avoid changing the csproj file unless required. Can you check over why these changes were made and perhaps reverse some of them

Reply:
Addressed in next commit, only thing required to keep was global suppression inclusion which seemed to be removed previously by mistake. (it simply covers the 'iOS' case)